src/ros_traj.kl

```diff
@@ -129,6 +130,7 @@ VAR
 
 	msg000B_               : sm000B_t
 	sm000B_lst_            : sm000B_lst_t   -- list of JOINT_TRAJ_PT msgs
+	outs_msgs_             : INTEGER
```

Comment:
Could you please add a comment here?

Something like: `-- number of messages received and to ACK`